api/src/models/JobMetadata.ts

```diff
@@ -25,6 +29,22 @@ class JobMetadata {
         }
     }
 
+    ingestLockfile(job) {
+        return job.dir + '/ingest.lock';
+    }
+
+    get raw() {
+        return {
+            order: this.order.raw,
+            published: this.published === true
+        };
+    }
+
+    set raw(data) {
+        this._order = PageOrder.fromRaw(data);
+        this._published = true;
```

Comment:
The published property is already a boolean, so I think you can just stick it here directly; no need for the extra `=== true`.